apps/dashboard/src/components/workflow-editor/steps/configure-step-content.tsx

```diff
@@ -0,0 +1,82 @@
+import { useMemo } from 'react';
+import { Link } from 'react-router-dom';
+import { RiArrowRightSLine, RiArrowRightUpLine, RiPencilRuler2Fill } from 'react-icons/ri';
+import { StepTypeEnum } from '@novu/shared';
+
+import { Button } from '../../primitives/button';
+import { Separator } from '../../primitives/separator';
+import { CommonFields } from './common-fields';
+import { SidebarContent } from '@/components/side-navigation/Sidebar';
+import { ConfigureInAppPreview } from './configure-in-app-preview';
+import { SdkBanner } from './sdk-banner';
+import { useStep } from './use-step';
+import { getFirstControlsErrorMessage, getFirstBodyErrorMessage } from '../step-utils';
+
+export const ConfigureStepContent = () => {
```

Comment:
This is the new component that I extracted from the `Step` component in the `ConfigureStep`.
The Configure Step sidebar content should not be much different for different step types, no need to overcomplicate it for now.